src/main.cpp

```diff
@@ -80,35 +80,63 @@ StartSubCommand CheckStartSubCommands(const wchar_t* arg)
 
 
 int ParseStopOrAnalyze(int argc, wchar_t* argv[], const wchar_t* command, const wchar_t* sessionOrInputHelp,
-    std::wstring& firstArg, std::wstring& outputFile, bool& analyzeTemplates)
+    std::wstring& firstArg, std::wstring& outputFile, bool& analyzeTemplates, bool& generateTimeTrace)
 {
     if (argc < 4)
     {
-        std::wcout << L"vcperf.exe " << command << " [/templates] " << sessionOrInputHelp << " outputFile.etl" << std::endl;
+        std::wcout << L"vcperf.exe " << command << " [/templates] " << sessionOrInputHelp << " [/timetrace] outputFile.etl" << std::endl;
         return E_FAIL;
     }
 
     int curArgc = 2;
 
     analyzeTemplates = false;
-    std::wstring arg = argv[curArgc++];
-    firstArg = arg;
+    generateTimeTrace = false;
+
+    // options prior to input file
 
+    std::wstring arg = argv[curArgc++];
     if (CheckCommand(arg, L"templates"))
     {
-        firstArg = argv[curArgc++];
         analyzeTemplates = true;
+        arg = argv[curArgc++];
+
+        if (argc < 5)
+        {
+            std::wcout << L"vcperf.exe " << command << " [/templates] " << sessionOrInputHelp << " [/timetrace] outputFile.etl" << std::endl;
```

Comment:
Two lines here as well. Maybe this printing code could be extracted to a function or lambda instead of repeating it.